AIM/sourcecode/CouplingFunctions.py

```diff
@@ -436,7 +436,7 @@ def CalcTCC_nb(AmGroupi, AmGroupj, WS_positions, AllAmGroups, halfbox, boxdims,
                   * AIM_MF.dotprod(vb, d)
                   * AIM_MF.dotprod(va, d))
             J -= ir3 * (dqi[a] * qj[b] * AIM_MF.dotprod(vb, d)
```

Comment:
I think that this is now correct even it may be a bit confusing compared to Eq. A6 of the Jansen 2006 paper as the distance vector is opposite to these and the sign of the two terms is reversed. Maybe it would be good to add a comment on this or use the same sign and direction as in the paper. It would maybe be good to add a reference in the comments to equation A6 of ref. 10.1063/1.2218516.

I looked up [AIM](https://github.com/Kimvana/AIM/tree/Version-1.0-(main))/[AIM](https://github.com/Kimvana/AIM/tree/Version-1.0-(main)/AIM)/[sourcefiles](https://github.com/Kimvana/AIM/tree/Version-1.0-(main)/AIM/sourcefiles)
/Map-C_TCC.dat
There the sign of the dq parameters are as in the paper, so the original direction given here was actually the correct one according to the erratum.